### PR TITLE
committing sample

### PR DESCRIPTION
**Pull request by jesusarockias**

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[0923/123721.035:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
```
Comment (ajithr) [inline debug.log]: Is this file needed?

File: scripts/debug.log [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[0923/123649.484:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
```
Comment (ajithr) [inline scripts/debug.log]: Is this file needed?

File: src/debug.log [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[0923/123825.346:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
```
Comment (ajithr) [inline src/debug.log]: Is this file needed?

File: debug.log (hunk 1 above)
Comment (jesusarockias) [reply debug.log]: I will check and remove if it is not needed

File: scripts/debug.log (hunk 2 above)
Comment (jesusarockias) [reply scripts/debug.log]: I will check and remove if not needed.
